components/inlinealert/index.css

```
@@ -120,42 +193,24 @@
 
 .spectrum-InLineAlert-footer {
 	display: flex;
-	justify-content: flex-end;
-	margin-block-start: var(--mod-inlinealert-spacing-header-content-button, var(--spectrum-inlinealert-spacing-header-content-button));
+	justify-content: flex-start;
+	margin-block-start: var(--mod-inlinealert-spacing-content-link-button, var(--spectrum-inlinealert-spacing-content-link-button));
 
 	&:empty {
 		display: none;
 	}
 }
 
-.spectrum-InLineAlert--info {
-	border-color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-info, var(--spectrum-inlinealert-border-and-icon-color-info)));
-
-	.spectrum-InLineAlert-icon {
-		color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-info, var(--spectrum-inlinealert-border-and-icon-color-info)));
-	}
-}
-
-.spectrum-InLineAlert--notice {
-	border-color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-notice, var(--spectrum-inlinealert-border-and-icon-color-notice)));
-
-	.spectrum-InLineAlert-icon {
-		color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-notice, var(--spectrum-inlinealert-border-and-icon-color-notice)));
-	}
-}
-
-.spectrum-InLineAlert--positive {
-	border-color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-positive, var(--spectrum-inlinealert-border-and-icon-color-positive)));
-
-	.spectrum-InLineAlert-icon {
-		color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-positive, var(--spectrum-inlinealert-border-and-icon-color-positive)));
-	}
-}
-
-.spectrum-InLineAlert--negative {
-	border-color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-negative, var(--spectrum-inlinealert-border-and-icon-color-negative)));
+@media (forced-colors: active) {
+	.spectrum-InLineAlert {
+		--highcontrast-inlinealert-header-color: CanvasText;
+		--highcontrast-inlinealert-content-color: CanvasText;
+		--highcontrast-inlinealert-border-and-icon-color: CanvasText;
 
-	.spectrum-InLineAlert-icon {
-		color: var(--highcontrast-inlinealert-border-and-icon-color, var(--mod-inlinealert-border-and-icon-color-negative, var(--spectrum-inlinealert-border-and-icon-color-negative)));
+		&.spectrum-InLineAlert--subtle,
+		&.spectrum-InLineAlert--bold {
+			--highcontrast-inlinealert-background-color: Canvas;
+			--highcontrast-inlinealert-border-width: var(--spectrum-inlinealert-border-width);
+		}
```

```suggestion
		--highcontrast-inlinealert-background-color: Canvas;

		&.spectrum-InLineAlert--subtle,
		&.spectrum-InLineAlert--bold {
			--highcontrast-inlinealert-border-width: var(--spectrum-border-width-200);
		}
```

1. The border isn't appearing for these variants right now because the value of the width variable is 0px (for these variants in not high contrast).
2. I'd move the background declaration up so the high contrast background is always applied; it doesn't appear to be limited to just these variants anymore.